install-devstack-xen.sh

```diff
@@ -4,11 +4,12 @@ set -eu
 function print_usage_and_die
 {
 cat >&2 << EOF
-usage: $0 XENSERVER XENSERVER_PASS PRIVKEY <optional arguments>
+usage: $0 XENSERVER XENSERVER_PASS PRIVKEY NODE_TYPE NODE_NAME [CONTROLLER_IP] <optional arguments>
```

Comment:
To preserve compatibility, can NODE_TYPE, NODE_NAME and CONTROLLER_IP be added as optional arguments instead of required arguments?